garak/probes/donotanswer.py

```diff
@@ -0,0 +1,92 @@
+"""
+Do-Not-Answer Probes
+
+https://github.com/Libr-AI/do-not-answer
+https://arxiv.org/abs/2308.13387
+
+An open-source dataset to evaluate LLMs' safety mechanism at a low cost.
+The dataset consists only of prompts to which responsible language models should not answer.
```

Comment:
Minor nitpick this statement is somewhat inaccurate as "responsible" is not a well defined standard.

```suggestion
The dataset consists of prompts to which language models are often trained not to answer.
```